README.md

```diff
@@ -33,7 +33,7 @@ Please check out https://github.com/bbernhard/signal-cli-rest-api#getting-starte
 ```bash
 docker run -p 8080:8080 \
     -v $(pwd)/signal-cli-config:/home/.local/share/signal-cli \
-    -e 'MODE=normal' bbernhard/signal-cli-rest-api:0.57
+    -e 'MODE=normal' bbernhard/signal-cli-rest-api:0.92
```

Comment:
How about setting this to latest, so that we don't have to keep updating it?
```suggestion
    -e 'MODE=normal' bbernhard/signal-cli-rest-api:latest
```